pkg/apis/config/defaults.go

```diff
@@ -52,6 +58,10 @@ func NewDefaultsConfigFromMap(data map[string]string) (*Defaults, error) {
 		}
 	}
 
+	if nc.RevisionTimeoutSeconds > nc.MaxRevisionTimeoutSeconds {
+		return nil, errors.New("RevisionTimeoutSeconds cannot be greater than MaxRevisionTimeoutSeconds")
```

Comment:
As this is configmap validation, should these actually contain the configmap-keys?

```
RevisionTimeoutSeconds -> revision-timeout-seconds
```